proto/ignition/msgs/parameter.proto

```diff
@@ -0,0 +1,34 @@
+/*
+ * Copyright (C) 2021 Open Source Robotics Foundation
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+*/
+
+syntax = "proto3";
+
+package ignition.msgs;
+option java_package = "com.ignition.msgs";
+
+/// \brief Representation of a parameter
+message Parameter
+{
+  /// \brief Parameter name;
+  string name = 1;
+
+  /// \brief Component type name;
+  string type = 2;
```

Comment:
do values in this field correspond to the `type` field in `ParameterDeclaration`? If so, I would make sure that the doc-strings match each other more closely.

Reply:
Docstrings updated in https://github.com/ignitionrobotics/ign-msgs/pull/211/commits/0b8692293256a4804060d393ced5836d8bb3ad1a.